python_modules/libraries/dagster-dg/dagster_dg/cli/plus.py

```diff
@@ -63,3 +74,128 @@ def login_command() -> None:
         url=config.url,
     )
     config.write()
+
+
+def _create_temp_deploy_dockerfile(dst_path, python_version):
+    dockerfile_template_path = os.path.join(
+        os.path.dirname(__file__), "..", "templates", "deploy_uv_Dockerfile.jinja"
+    )
+
+    loader = jinja2.FileSystemLoader(searchpath=os.path.dirname(dockerfile_template_path))
+    env = jinja2.Environment(loader=loader)
+
+    template = env.get_template(os.path.basename(dockerfile_template_path))
+
+    with open(dst_path, "w", encoding="utf8") as f:
+        f.write(template.render(python_version=python_version))
+        f.write("\n")
+
+
+@plus_group.command(name="deploy", cls=DgClickCommand)
+@click.option(
+    "--organization",
+    "organization",
+    help="Dagster+ organization to which to deploy. If not set, defaults to the value set by `dg plus login`.",
+    envvar="DAGSTER_PLUS_ORGANIZATION",
```

Comment:
I think we probably want centralized tooling to merge flags / env vars / plus config, and loudly tell the user if env vars are taking precedence over the file (at some point, not in this PR)